src/lib.rs

```diff
@@ -26,17 +26,6 @@ use std::task::{Context, Poll};
 /// The `poll_trailers` function returns an optional set of trailers used to finalize the request /
 /// response exchange. This is mostly used when using the HTTP/2.0 protocol.
 ///
-/// # Relation with [`BufStream`].
-///
-/// The `Body` trait is a superset of the `BufStream` trait. However, `BufStream` is not considered
-/// a super trait of `Body`. Instead, a `T: Body` can be thought of as containing a `BufStream` as
-/// well as the HTTP trailers.
-///
-/// There exists is a blanket implementation of `Body` for `T: BufStream`. In other words, any type
-/// that implements `BufStream` also implements `Body` yielding no trailers.
-///
-/// [`BufStream`]: https://docs.rs/tokio-buf/0.1.1/tokio_buf/trait.BufStream.html
-///
```

Comment:
If cleaning up the docs, I noticed there's other mentions of `BufStream`, like `Body::Error` and in `SizeHint` docs.